clusterloader2/testing/load/config.yaml

```diff
@@ -249,6 +252,7 @@ steps:
     - basename: big-deployment
       objectTemplatePath: deployment.yaml
       templateFillMap:
+        EnableDNSTests: {{$ENABLE_DNSTESTS}}
```

Comment:
If you set EnableDNSTests to {{$ENABLE_DNSTESTS}} in big-deployment, medium-deployment and small-deployment you will get ~150k pods running queries (all pods will be running queries).

As we agreed on chat, let's start with 37k first (so only medium-deployment) and we will see if this works.

In order to do that, you need to set EnableDNSTests to false in both big-deployment and small-deployment in deployment.yaml and configmap.yaml.
Please also add a comment like "We are setting this to false in big and small to enable this only in medium", as it's not obvious for random reader why we are copying this value only for medium".

Reply:
Thanks! Done.